dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -390,6 +420,79 @@ class BGPPeerMgr(Manager):
         return peers
 
 
+def prefix_attr(attr, value):
```

Comment:
this is really a *bad* interface design! the function name is generic. in fact the fuction does very specific thing, it assume the value is ip prefix and convert the string into ipnetwork, and then get an attribute, and finally convert it into a string. 

to evaulate whether a function provide a good abstraction or not. People should be able to understand the what the function's job by just looking at the function name at the caller without looking into the implementation of the function. 

I think you need to either get rid of this function, or have a function get_ip_from_ipprefix function to do the specific job you want.

Reply:
Yes, I agree with your comment. And I think this is a very common tool function maybe we should extract it to a common python package.
Because we did also use it at : https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/sonic-cfggen#L81 
Should I do that?

Reply:
you are right, that was my miss. it is not a good code.

---
In reply to: [400696041](https://github.com/Azure/sonic-buildimage/pull/4260#discussion_r400696041) [](ancestors = 400696041)